css/main.css

```diff
@@ -0,0 +1,54 @@
+body {
+	background: black;
+	color: white;
+	display: flex;
+	justify-content: center;
+	background-image: url(http://i.kinja-img.com/gawker-media/image/upload/msykmohxox4buoeuvp2y.jpg); 
+}
+
+#instructions {
+	display: flex;
+	justify-content: center;
+	margin-top: 300px;
+}
+
+p {
+	font-family: Arial;
+	font-size: 3em;
+}
+
+#photos  {
+	position: absolute;
+	top: 30px;
+	left: 500px;
+
```

Comment:
Using `position: absolute` was overlaying the image on top of the button. It'd be better to use `flexbox` or `block`, and place the `#photos` div below the rest of the content.